src/platform/amd/renoir/include/platform/drivers/interrupt.h

```diff
@@ -26,13 +26,13 @@
 
 #define IRQ_NUM_TIMER0		0	/* level 1 */
 
-#define IRQ_NUM_EXT_LEVEL3	3	/* level 1 */
+#define IRQ_NUM_EXT_LEVEL3	3	/* level 3 */
 
-#define IRQ_NUM_TIMER1		6	/* level 2 */
+#define IRQ_NUM_TIMER1		6	/* level 5 */
 
-#define IRQ_NUM_EXT_LEVEL4	4	/* level 2 */
+#define IRQ_NUM_EXT_LEVEL4	4	/* level 4 */
 
-#define IRQ_NUM_EXT_LEVEL5	5	/* level 3 */
+#define IRQ_NUM_EXT_LEVEL5	5	/* level 5 */
 
```

Comment:
This is just fixing some comments and needs to be added in a separate patch. Remember one patch per logical unit change.

Reply:
Removed the comment change from the patch.